docker/api/client.py

```diff
@@ -1,10 +1,15 @@
 import json
 import struct
 import urllib
+import ssl
 from functools import partial
+from typing import Any, AnyStr, Optional, Union, Dict, overload, NoReturn, Iterator
+
+from typing_extensions import Literal
```

Comment:
Better to only require typing_extensions on older Python versions.
```suggestion
if sys.version_info >= (3, 8):
    from typing import Literal
else:
    from typing_extensions import Literal
```